app/models/tweeter.rb

```diff
@@ -48,21 +42,9 @@ def bounty_participants(bounty)
     participants.uniq.take(3)
   end
 
-  def bounty_hashtags(bounty)
-    QueryMarks.new.legible_mark_vector(bounty.mark_vector).sort_by{|a, b| -b}.map{|a, b| a}.take(4)
-  end
-
-  def top_products_by_activity
-    ProductStats.top_products_by_activity(limit: 12).select{|a, b| a != "meta"}.map{|a, b| Product.find_by(slug: a)}
-  end
-
-  def bounties_from_top_products
-    top_products_by_activity.map{|a| a.tasks}.flatten.select{|a| a.state == "open"}
-  end
-
   def worthy_bounties(n)
     bounty_suggestion_fraction = 0.5
-    sorted_top_bountys = bounties_from_top_products.select{|a| a.earnable_coins_cache}.sort_by{|a| -a.earnable_coins_cache}
+    sorted_top_bountys = TweetPrep.bounties_from_top_products.select{|a| a.earnable_coins_cache}.sort_by{|a| -a.earnable_coins_cache}
```

Comment:
Line is too long. [133/80]<br>Space missing to the left of {.<br>Space between { and | missing.<br>Space missing inside }.<br>Pass `&:earnable_coins_cache` as an argument to `select` instead of a block.